sysstat_report.py

```diff
@@ -470,6 +474,12 @@ class Plotter:
             "ylabel": "Activity (MB/s)",
             "yrange": (0, None),
         },
+        SysstatDataType.DISK: {
+            "title": "Disks",
+            "data_titles": ("%",),
+            "ylabel": "DISK usage (%)",
```

Comment:
See first review comment, I would prefer the more generic "Filesystem" naming instead of "Disk".
So "Filesystem usage" for `title`, and just "Usage (%)" for the `ylabel`.